automatic_spinnaker_make.sh

```diff
@@ -0,0 +1,29 @@
+# This script assumes it is run from the directory holding all github projects in parellel
+# sh SupportScripts/automatic_make.sh
+
```

Comment:
You should add this to the script at this point:
```
set -e
```
It makes the shell do automatic error handling; it exits on any error and that's probably what you want for this whole script. (It makes errors work a lot like exceptions.)